app/src/main/java/org/schabi/newpipe/about/StandardLicenses.kt

```diff
@@ -0,0 +1,13 @@
+package org.schabi.newpipe.about
+
+/**
+ * Class containing information about standard software licenses.
+ */
+object StandardLicenses {
+    @JvmField
```

Comment:
Why is `@JvmField` only prefixed to the first one?

Reply:
Oops, lol. That was the automatic conversion. I'll fix it.